### PR TITLE
Build out README and CONTRIBUTING docs

### PR DESCRIPTION
**Pull request by nekevss**
This PR builds out the README a tad more and adds a CONTRIBUTING.

Let me know what you think about the wording, especially around the CONTRIBUTING doc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+If you're interested in helping out but don't see an issue that's for you, please feel free to contact
+us on the `Boa` discord.
+
+## Contributor Information
+
+`Temporal` is a new date/time API that is being developed and proposed for the `ECMAScript`
+specification. This library aims to be a `Rust` implementation of that specification.
+
+Due to the current experimental nature of the material and this library, we would ask that
+potential contributors familiarize themselves with the specification.
```
Comment (jedel1043) [inline CONTRIBUTING.md:19]:
```suggestion
Due to the current experimental nature of the material and this library, we would
advise potential contributors to familiarize themselves with the Temporal specification.
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+builtin to the `ECMAScript` specification.
 
-## Goal
+This crate is an implementation of `Temporal` in Rust. While initially developed for the `Boa`, the crate has been externalized
+as we intended to make an engine agnostic and general usage implementation of `Temporal` and its algorithms.
 
-The intended goal of this crate is to provide an engine agnostic
-implementation of `ECMAScript`'s Temporal algorithms.
+IMPORTANT NOTE: The Temporal Proposal is still a Stage 3 proposal. As such, regardless of current state of this repository,
+please refrain from using without flags.
```
Comment (jedel1043) [inline README.md:12]: I think all the warnings about the experimental nature of the crate can be joined into a single warning at the start of the README.

**Review by jedel1043 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Contributing to Temporal in Rust
+
+This project is currently highly experimental and fairly volitile. As such, while we do
+welcome contributions, we'd like to ask that you reach out on an issue first to ensure
+that that feature is not currently being actively worked on, and so that we can create
+a tracking issue (if needed) and assign that issue to you.
+
+We'd like to kindly ask to refrain from any unsolicited or unassigned Pull Requests at this time.
```
[the remark this answers is not in the document]
Comment (nekevss) [reply CONTRIBUTING.md:8]: Yeah, the wording is a bit too harsh.

**Review by jedel1043 — APPROVED** [review]
Nice documentation!

**Pull request merged**